crates/commonware-node/src/dkg/manager/ingress.rs

```diff
@@ -29,31 +29,28 @@ impl Mailbox {
     ///
     /// Returns `None` if this node was not a dealer, or if the request is
     /// for a different epoch than the ceremony that's currently running.
-    pub(crate) async fn get_intermediate_dealing(
+    pub(crate) async fn get_dealer_log(
         &self,
         epoch: Epoch,
-    ) -> eyre::Result<Option<IntermediateOutcome>> {
+    ) -> eyre::Result<Option<SignedDealerLog<MinSig, PrivateKey>>> {
         let (response, rx) = oneshot::channel();
         self.inner
-            .unbounded_send(Message::in_current_span(GetIntermediateDealing {
-                epoch,
-                response,
-            }))
+            .unbounded_send(Message::in_current_span(GetDealerLog { epoch, response }))
             .wrap_err("failed sending message to actor")?;
         rx.await
-            .wrap_err("actor dropped channel before responding with ceremony deal outcome")
+            .wrap_err("actor dropped channel before responding with signed dealer log")
     }
 
-    pub(crate) async fn get_public_ceremony_outcome(
+    pub(crate) async fn get_dkg_outcome(
         &self,
-        parent: (View, Digest),
-        round: Round,
-    ) -> eyre::Result<PublicOutcome> {
+        digest: Digest,
```

Comment:
super nit: can we call these things `block_hash` or `parent_hash` instead of `digest`? i was lost on what this `digest` referred to when first glancing at it